addons/l10n_ch/views/res_bank_view.xml

```diff
@@ -9,6 +9,8 @@
                 <xpath expr="//field[@name='acc_number']" position="after">
                     <field name="l10n_ch_qr_iban" invisible="not l10n_ch_display_qr_bank_options"/>
                     <field name="l10n_ch_display_qr_bank_options" invisible="1"/>
+                    <!-- invisible field added because it is used as a conditional
+                    expression for the visibility of l10n_ch_qr_iban -->
```

Comment:
```suggestion
```
If I understood the [PR](https://github.com/odoo/odoo/pull/137031)'s purpose correctly, we should not need to keep this field in the view, it will be cached automatically by the ORM, right?

![image](https://github.com/user-attachments/assets/1110fd4e-edff-4c87-a1f4-8376dd6a46fa)